src/main/java/org/folio/rest/impl/OrdersImpl.java

```diff
@@ -222,6 +225,18 @@ public void putOrdersLinesByIdAndLineId(String orderId, String lineId, String la
     }
   }
 
+  @Override
+  @Validate
```

Comment:
The `po_number.json` schema defines `"pattern": "^[a-zA-Z0-9]{5,16}$"` and `@Validate` annotation should let the RMB to validate `PoNumber` content. Looks like `HelperUtils.isPOValid` is not required in this case...

Reply:
Yea that method is not being used anywhere currently, I just left it in case if has to be used by the other APIs